plugins/lime-plugin-fbw/src/containers/NetworkForm.js

```diff
@@ -12,48 +12,72 @@ import I18n from 'i18n-js';
 import { isValidHostname, slugify } from '../../../../src/utils/isValidHostname';
 
 export const NetworkForm = ({ createNetwork, toggleForm }) => {
-	const [ state, setState ] = useState({
+	const [state, setState] = useState({
 		communityName: '',
 		hostName: '',
-		password: ''
+		password: '',
+		passwordConfirmation: '',
 	});
 
-	function _changeName (e) {
+	function _changeName(e) {
 		const end = e.type === 'change';
 		e.target.value = slugify(e.target.value.toLocaleLowerCase(), end, true);
 		setState({ ...state, communityName: e.target.value || '' });
 		return e;
 	}
 
-	function _changeHostName (e) {
+	function _changeHostName(e) {
 		const end = e.type === 'change';
 		e.target.value = slugify(e.target.value, end);
 		setState({ ...state, hostName: e.target.value || '' });
 		return e;
 	}
 
-	/* function _changePassword (e){
-		setState({ ...stat, password: e.target.value || '' });
-	} */
+	function _changePassword(e) {
+		setState({ ...state, password: e.target.value || '' });
+		return e;
+	}
+
+	function _changePasswordConfirmation(e) {
+		setState({ ...state, passwordConfirmation: e.target.value || '' });
+		return e;
+	}
 
-	function _createNetwork(){
-		createNetwork({ network: state.communityName, hostname: state.hostName });
+	function _createNetwork() {
+		createNetwork({
+			network: state.communityName,
+			hostname: state.hostName,
+			password: state.password,
+		});
 		toggleForm('setting')();
 	}
 
+	function _isValidPassword() {
+		return state.password.length > 0;
```

Comment:
password length should be at least 10 to be valid, and with at least numbers and letters.

Reply:
password validation is now implemented